docs/user-guide/nextcloud-integration/README.md

```diff
@@ -124,13 +188,6 @@ When a Nextcloud file or folder is linked to a work package, an OpenProject user
 However, all available actions depend on permissions the OpenProject user (or more precisely, the Nextcloud account tied to that user) has in Nextcloud. In other words, a user who does not have the permission to access the file in Nextcloud will also *not* be able to open, download, modify or unlink the file in OpenProject.
 
 
-
-## **Next steps for the integration**
```

Comment:
@psatyal and @wielinde do we want to explain what it might come in 13.0 to the users?

Reply:
I think it made sense in the last version since uploading from OP was an obvious omission that users would surely notice or wonder about. I'm removing this section because generally, documentation does not aim to inform users about upcoming changes. This feature is quite complete (from the user's pov).